e2e-tests/playwright/e2e/plugins/rbac/rbac-api.spec.ts

```diff
@@ -267,8 +303,10 @@ test.describe("Test RBAC plugin REST API", () => {
         responseHelper.getSimpleRequest(),
       );
 
-      expect(deleteRemainingPolicies.ok());
-      expect(deleteRole.ok());
-    },
-  );
+      expect(deleteRemainingPolicies.ok()).toBeTruthy();
+      expect(deleteRole.ok()).toBeTruthy();
+    } catch (error) {
+      console.error("Error during cleanup in afterAll:", error);
```

Comment:
console.error will not report test failure, it will give false positive afaik.

Reply:
You’re right. I will replace console.error with assertions or other mechanisms that correctly fail the test if cleanup encounters issues. Thank you for pointing that out!